### PR TITLE
Translated Readme file to Hebrew

### PR DESCRIPTION
**Pull request by omer-dor**
Task 1/4 - issue #18

### CODE REVIEW

**Comment by omarespejel** [comment]
Thanks @omer-dor!

File: tutorials/HEB/01_huggingface-hub-tour.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+# Workshop: A Tour through the Hugging Face Hub
```
Comment (omarespejel) [inline tutorials/HEB/01_huggingface-hub-tour.md:1]: I think we can eliminate this file until we have a Hebrew translation.

File: tutorials/HEB/02_ml-demos-with-gradio.ipynb [hunk 2]
```diff
@@ -0,0 +1,497 @@
+{
```
Comment (omarespejel) [inline tutorials/HEB/02_ml-demos-with-gradio.ipynb:1]: We can also eliminate this document until we have the Hebrew translation.

File: tutorials/HEB/03_getting-started-with-transformers.ipynb [hunk 3]
```diff
@@ -0,0 +1,9726 @@
+{
```
Comment (omarespejel) [inline tutorials/HEB/03_getting-started-with-transformers.ipynb:1]: We can also eliminate this document until we have the Hebrew translation.

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thank you @omer-dor! Only some nits.

**Comment by omer-dor** [comment]
@omarespejel I deleted the files you requested, I will translate them also next week and create another PR.

**Review by omarespejel — APPROVED** [review]
This is great! Thanks 🤗

**Comment by omarespejel** [comment]
Merged. Thanks! Please let me know if you need anything.

**Pull request merged**